src/client/components/RegistrationConsents.tsx

```diff
@@ -23,48 +26,54 @@ const consentToggleCss = (noMarginBottom = false) => css`
 	gap: ${space[3]}px;
 `;
 
+const chooseNewsletter = (
+	geolocation: GeoLocation | undefined,
+	appName: AppName | undefined,
+): RegistrationNewsletterFormFields | undefined => {
+	const isFeast = appName === 'Feast';
+
+	if (isFeast) {
+		return RegistrationNewslettersFormFieldsMap.feast;
+	}
+
+	switch (geolocation) {
+		case 'GB':
+		case 'EU':
+		case 'ROW':
+			return RegistrationNewslettersFormFieldsMap.saturdayEdition;
+		case 'US':
+			return RegistrationNewslettersFormFieldsMap.usBundle;
+		case 'AU':
+			return RegistrationNewslettersFormFieldsMap.auBundle;
+		default:
+			// We want to show Saturday Edition even for an undefined location
+			return RegistrationNewslettersFormFieldsMap.saturdayEdition;
+	}
```

Comment:
does one of the following read better?

```suggestion
	switch (geolocation) {
		case 'US':
			return RegistrationNewslettersFormFieldsMap.usBundle;
		case 'AU':
			return RegistrationNewslettersFormFieldsMap.auBundle;
		default:
			// We want to show Saturday Edition even for anywhere else (including 'GB', 'EU', 'ROW')
			return RegistrationNewslettersFormFieldsMap.saturdayEdition;
	}
```

or

```suggestion
	switch (geolocation) {
		case 'US':
			return RegistrationNewslettersFormFieldsMap.usBundle;
		case 'AU':
			return RegistrationNewslettersFormFieldsMap.auBundle;
		case 'GB':
		case 'EU':
		case 'ROW':
		default:
			// We want to show Saturday Edition even for an undefined location
			return RegistrationNewslettersFormFieldsMap.saturdayEdition;
	}
```

Reply:
I like the bottom one because it explicitly calls out all the options for `geolocation`.